playground/AspireEventHub/EventHubs.AppHost/aspire-manifest.json

```diff
@@ -27,6 +28,7 @@
       "env": {
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
+        "OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY": "in_memory",
```

Comment:
Unrelated?

Reply:
This manifest had not been regenerated since that change was made. I regenerated the manifests as part of this PR to keep them up to date. I uploaded all the playground manifests into the schemastore build system as postive test cases.